atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/AsyncCellLoader.java

```diff
@@ -82,12 +84,15 @@ private AsyncCellLoader(CqlClient cqlClient, Executor executor, String keyspace)
             long timestamp) {
         return cqlClient.executeQuery(
```

Comment:
Can we add some unit tests for this class in general?

Reply:
In fact I see tests only for QueryCache, so we very likely should have more.

Reply:
Added unit tests for AsyncCell loader. They are based on around weather visibility is maintained. Integration tests for CKVS should cover if the `getAsync` behaves correctly.